hello/test/test.c

```diff
@@ -4,23 +4,21 @@
 #include <assert.h>
 #include "../src/hello.h"
 
-int
-main(void) {
+int main(void) {
 
   char *foo = hello("foo");
   char expected[12] = "Hello: foo\n";
 
   if (strcmp(foo, expected)) {
-    printf("\n");
+    puts("\n");
```

Comment:
puts("\n") should be puts("");

puts() output line feed code at last.
http://www.c-tipsref.com/reference/stdio/puts.html